### ISSUE TITLE
Application Damaged Error for macOS (arm) Binary

### ISSUE THREAD

**Issue by Cyano-Hibiki** [poooi/poi#2401]
**poi 版本 / poi version:** 10.7.0

**操作系统 / OS:** macOS 12.0.1 (21A559)

**你遇到了什么样的问题 / The problem you've met:** 
I have a MacBook Pro with Apple M1 Pro chip. The image **poi-10.7.0-arm64.dmg** was downloaded from poi.moe. After dragging the application into Application folder and run, an error pops up: **“poi.app” is damaged and can’t be opened. You should move it to the Trash." 

**有没有重现的方法，或者与问题相关的任何信息 / How to reproduce, or any information that might be related:** 
To reproduce, download the Image **poi-10.7.0-arm64.dmg** from poi.moe, install it, and attempt to run the application. I tried to download directly from GitHub, but the error persists. The 64-bit macOS version will run but is extremely laggy, given how well many other 64-bit apps run over Rosetta 2.

**Comment by gnattu** [comment]
You want to use `xattr -d com.apple.quarantine /Applications/poi.app` in console, the arm64 binary is not correctly signed (atm) and macOS is refusing to launch it.

**Comment by Cyano-Hibiki** [comment]
Thank you. My issue is resolved after running the command in console. With that said, I think it might be a good idea to (temporarily) mention this caveat somewhere on README.md page and your official site (or can simply refer to this issue), before the the signing issue is resolved.

**Comment by KagamiChan** [comment]
Thanks for the advice, we're considering signing our builds for macOS

**Comment by KagamiChan** [comment]
Hi @Cyano-Hibiki
here's our signed builds, could you please give it a try if interested?

**Comment by Cyano-Hibiki** [comment]
Hello @KagamiChan,

Thank you for providing the signed builds. 

I tried it out, and it worked. Below are the steps I took, for those who are reading this issue thread later.

- I downloaded "**poi-10.7.0-arm64-mac.zip**". (Before installing this new binary, I used **AppCleaner** to fully remove previous version.)
- The signed application no longer report the Application Damage Error. Instead, it reports a more common error "**Apple cannot check it for malicious software**" error.
- This one can be overridden by going into **System Preferences** -> **Security and Privacy** -> "**General**" and click on “**Open Anyway**”.
- After that the application should launch without any error. On the first successful launch there might be an issue where the UI would not show up (unable to see tabs and other information). I closed the app and relaunched it, and the issue no longer persisted.

I shall keep using this build for a while. Should any other error show up, I will open a new issue thread.
